siliconcompiler/core.py

```diff
@@ -2997,7 +2998,7 @@ def summary(self, steplist=None, show_all_indices=False,
 
             # Try to open the results and layout only if '-nodisplay' is not set.
             # Prioritize PDF, followed by HTML.
-            if (not self.get('option', 'nodisplay')) and ('DISPLAY' in os.environ):
+            if (not self.get('option', 'nodisplay')):
```

Comment:
I think you still need this check for linux. Maybe the thing to do is to check that the os is windows or DISPLAY in os.environ